### DISCUSSION TITLE
Permissions to access mount folder

### DISCUSSION THREAD

**Discussion by iannyx** [termux/termux-app#2306] [Q&A] [answered]
Hi there, I have a Nexus7 (2012 - wifi) with Lineage-14.1 (20210313-UNOFFICIAL-grouper) and it is rooted (addonsu-14.1-arm-signed). I have been able to mount my NAS/share-folder via Termux+BusyBox and for do that I had to use root. Result: user "u0_a69" cannot access the mount folder (permission denied) so no one daemon installed via pkg can have access to it. Despite I did chmod 777 to the folder and it display "drwxrwxrwx 1 root root" only Termux/root and all the users/app outside Termux can r+w NAS/share-folder mounted on Android... but I need that "u0_a69" could be able to access the folder too. Some lights to help me, please?

**Comment by ghost** [comment, marked as answer]
Few things to consider:
1. SELinux labels - make sure this one is same as for shared storage to let external apps access it.
2. Mount namespace - `su` needs to work in "master" mount namespace, otherwise external apps won't see it.

Issue is not Termux-specific.

**Reply by iannyx** [reply]
Hi @xeffyr thanks for your replay,... and sorry for my delay.

Yes I checked and all things looks OK
Termux external users or apps can manage to access NAS folder mounted. No problems
Termux internal apps are installed by user "u0_a69" and NAS/share-folder is mounted by root so all internal apps will denied to access that folder.

However  the NAS/share-folder is set on 777 but  user "u0_a69" still has no access!

thanks
